src/index.html

```diff
@@ -0,0 +1,184 @@
+                <a href="https://wiki.fluidproject.org/display/fluid/%28Floe%29+Sonification">sonification framework</a>
+                that will help OER developers more easily create sound-based representations of interactive data
+                that can increase comprehension and reinforce learning.
+                </p>
+                <p>The <a href="http://build.fluidproject.org/chartAuthoring/demos/">Chart Authoring</a> tool demonstrates
+                how audio can enhance comprehension of data.
+                </p>
+            </div>
+        </div>
+            <!--
+            This framework is being used to add audio respresentations
+            of data to our <a href="http://build.fluidproject.org/chartAuthoring/demos/">Chart Authoring Tool</a>.</p>
+            -->
+
+        <div class="row">
+            <div class="columns large-6 small-12  large-order-1 small-order-2">
+                <div class="floe-image floe-image-preferenceIcons"></div>
+            </div>
+
+
+            <div class="columns large-6 small-12 large-order-2 small-order-1">
+                <h2>Need guidance in designing accessible resources?</h2>
+                <p>The <a href="http://handbook.floeproject.org/">Inclusive Learning Design Handbook</a>
+                is an expanding source of guidance, tools and tips for producing inclusively designed learning resources,
+                including guidelines for creating
+                <a href="http://handbook.floeproject.org/InclusiveEPUB3.html">inclusive EPUB 3 resources</a>,
+                <a href="http://handbook.floeproject.org/AccessibleStandardizedTesting.html">accessible standardized tests</a>, and
+                <a href="http://handbook.floeproject.org/WebGamesAndSimulations.html">inclusive web games and simulations</a>.</p>
+
+                <p>Floe has created <a href="http://guide.inclusivedesign.ca/">The Inclusive
+                Design Guide</a> - an engaging and evolving resource for learning about and applying Inclusive Design principles, practices,
+                and tools to the design process.
+                </p>
+
+                <p>The Metadata Editor makes it simple to provide useful labels regarding the learner preferences the resource can meet,
+                and supports authors in adding accessibility features.
+                The Feedback Tool allows users to help in the refinement of content and its metadata.</p>
+                <p>Floe has added a <a href="https://github.com/fluid-project/infusion-docs/blob/master/src/documents/TextToSpeechAPI.md">text-to-speech component</a>
+                to the Infusion JavaScript framework. The component uses browser-based speech synthesis
+                to provide self-voicing capabilities to web resources.</p>
+                <p>The<a href="https://github.com/fluid-project/metadata)">source code for the Metadata Editor and Feedback Tool</a>is available on Github.</p>
```

Comment:
Instead of a new paragraph, you can just add the text to the end of line 123.

Reply:
fixed